### DISCUSSION TITLE
Upgrade to latest mkdocstrings-python, or restrict Griffe to <0.33

### DISCUSSION THREAD

**Discussion by pawamoy** [mkdocstrings/python#98] [Announcements]
A new version of Griffe was released today, v0.33.0, and it is incompatible with mkdocstrings-python.

This was expected, and we released a mkdocstrings-python version that rejects it, v1.3.0, ten days ago. Please upgrade mkdocstrings-python to 1.3.0, or add a restriction on your accepted versions of Griffe in your dependency specs: `griffe<0.33`.

The next version of mkdocstrings-python (v1.4) will support Griffe 0.33.

**Comment by AlexWaygood** [comment]
Thanks for the heads-up!

**Comment by pawamoy** [comment]
mkdocstrings-python 1.4.0 is now up :slightly_smiling_face: 
Crossing fingers for no bugs :crossed_fingers:

**Reply by itdependsnetworks** [reply]
Addressed in https://github.com/mkdocstrings/python/pull/96 for future reference.

**Comment by pawamoy** [comment]
New version of Griffe again, 0.35.0. Please also upgrade mkdocstrings-python (to 1.5.1) if you upgrade Griffe.

See https://github.com/mkdocstrings/griffe/pull/202, https://github.com/mkdocstrings/python/pull/99 and for users with access to insiders, https://github.com/pawamoy-insiders/mkdocstrings-python/pull/1.
